AntennaTracker/servos.cpp

```diff
@@ -7,18 +7,19 @@
 // init_servos - initialises the servos
 void Tracker::init_servos()
 {
-    // setup antenna control PWM channels
-    channel_yaw.set_angle(g.yaw_range * 100/2);        // yaw range is +/- (YAW_RANGE parameter/2) converted to centi-degrees
-    channel_pitch.set_angle((-g.pitch_min+g.pitch_max) * 100/2);    // pitch range is +/- (PITCH_MIN/MAX parameters/2) converted to centi-degrees
+    SRV_Channels::set_default_function(CH_YAW, SRV_Channel::k_steering);
```

Comment:
I've added k_tracker_yaw and k_tracker_pitch. That means it will show up properly in the selection list in the GCS when assigning channels